src/shared-components/StandardHeaderDropdown/StandardHeaderDropdown.module.css

```
@@ -0,0 +1,72 @@
+  display: inline-flex;
+  align-items: center;
+}
+
+/* ── Toggle button (passed as toggleClassName) ────────────── */
+.toggleButton {
+  display: inline-flex;
+  align-items: center;
+  gap: var(--space-2);
+  padding: var(--space-2) var(--space-4);
+
+  background-color: var(--color-white);
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)